pages/Whiteboard___Object___Tweet.md

```diff
@@ -0,0 +1,7 @@
+alias:: Tweet, Tweets
+type:: [[Whiteboard/Object]]
```

Comment:
🔥 Great to see us adding more types/classes as they are definitely preferred over namespaces for organization. Like [other classes](https://docs.logseq.com/#/page/ui%20element), it'd be helpful to add `type:: [[Class]]` and a description